libs/libxx/libcxxabi.defs

```diff
@@ -55,6 +55,9 @@ else
 CPPSRCS += cxa_noexception.cpp
 endif
 
+# RTTI is required for building the libcxxabi library
+CXXFLAGS += -frtti
```

Comment:
@gustavonihei  libcxx/uClibc++ share the same CXXFLAGS with libcxxabi, this change will enable rtti for the standard library.

Reply:
You are right, I haven't noticed it before. I assumed that setting would be specific to `libcxxabi`.
NuttX build system supports file-specific compiler flags. Although functional, it seems rather ugly:
```suggestion
libcxxabi/src/private_typeinfo.cpp_CXXFLAGS += -frtti
```
The other way, and most correct I believe, would be to create a switch for RTTI support.
And this config could be applied to every board makefile, but it is not a small change hehe

Reply:
Another method is select rtti by CXX_EXCEPTION too.

Reply:
> Another method is select rtti by CXX_EXCEPTION too.

Do you mean to rely on the same switch? I think that's not very intuitive.

Another option would be to separate the ABI libs from the standard implementation libs.
This would allow us to to compile libcxxabi with RTTI.

Reply:
It's a good idea to separate ABI libs.